### ISSUE TITLE
~~ WARNING: Exodus 4D is AWOL - Read this before posting Issues/Pull Requests ~~

### ISSUE THREAD

**Issue by TyrHeimdalEVE** [exodus4d/pathfinder#1039]
The original developer of Pathfinder (Exodus 4D) seems to have abandoned the project.

There's no new commits since 2020-04-24, and there's been no information that this is going to change.

As such, there is a community-supported fork available (which you should use) over at [https://github.com/goryn-clade/pathfinder/](https://github.com/goryn-clade/pathfinder/).

If the issue you are facing persists when using the updated version, please submit a [new issue there](https://github.com/goryn-clade/pathfinder/issues).

You can also drop by the same [Slack Server](https://join.slack.com/t/pathfinder-eve-online/shared_invite/enQtMzMyOTkyMjczMTA3LWI2NGE1OTY5ODBmNDZlMDY3MDIzYjk5ZTljM2JjZjIwNDRkNzMyMTEwMDUzOGQwM2E3ZjE1NGEwNThlMzYzY2Y) as always to get help or interact with the community.

**Comment by jbuehlz** [comment]
@TyrHeimdalEVE Will your fork / org also be maintaining esi repo? There will be more unresolvable issues there, like that recent formData header.

**Comment by TyrHeimdalEVE** [comment]
@vorion 
Yes - we already do, and we'll probably move the fork of pathfinder_esi from https://github.com/tyrheimdaleve/pathfinder_esi to goryn-clade and maintain it there on next release (composer.json on the goryn-clade fork currently points to mine at packagist).

The issue you are referring to was already fixed in commit https://github.com/TyrHeimdalEVE/pathfinder_esi/commit/3c67432cfecadb3fbbcd886057e0afdcd86a6741 at the start of September.

There are other upcoming changes to SSO which are in the works to utilize JWT tokens etc.

We do not intend to "hijack" Pathfinder, but as we do not have any access to do changes on Exodus4d's repositories - we do not have much choice but continue it in a fork.

And when I say "we", I do mean the community. Hopefully we can drive it forward and see PR's from others who want to contribute as well.

**Comment by aechiara** [comment]
this is an easy fix, go to pathfinder directory, find the file `./vendor/exodus4d/pathfinder_esi/app/Config/Ccp/Esi/Config.php`
on line 41 where it reads `'GET' => '/v4/characters/{x}/',` change it to `'GET' => '/v5/characters/{x}/',`

should be enough
